keps/sig-windows/1122-windows-csi-support/README.md

```diff
@@ -768,12 +777,366 @@ Downgrading the version of csi-proxy.exe to one that is not supported by all ins
 
 Beyond the points in the above section (Upgrade/Downgrade strategy), there are no Kubernetes version skew considerations in the context of this KEP.
 
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+-->
+
+- [ ] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name:
+  - Components depending on the feature gate:
+- [x] Other
+  - Describe the mechanism:
+  CSI-proxy is a binary that can be deployed (or running as a service) directly on each Windows node during node start up time. For example, in gce, CSI-proxy
+  binary is downloaded and running as Windows service in node [startup script](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/k8s-node-setup.psm1#L424)
+  An environment variable ENABLE_CSI_PROXY is implemented to disable/enable CSI-proxy installation.
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+  No
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+  If csi-proxy binary is deployed as a binary or windows service during node start up time, enabling this feature will require reprovisioning of a node.
```

Comment:
Could someone install and launch csi-proxy without rebooting or creating a new node?

The reprovisioning of a node seems like a kube-up specific thing?